inc/knowbaseitem_revision.class.php

```diff
@@ -296,6 +299,7 @@ public function createNew(KnowbaseItem $item) {
       );
       $this->fields['date_creation'] = date('Y-m-d H:i:s');
       $this->fields['revision'] = $this->getNewRevision();
+      $this->fields['users_id'] = $item->fields['users_id'];
```

Comment:
Are you sure users_id in the KBitem is correct? I'd use user id from logged in user to be sure (the one from session)

Reply:
No, this piece of code acts on pre_update event to store current content as revision before updating it with input. So it will store in revision the user which writes the current content.

Reply:
OK then :)